docs/advanced/general-privacy.md

```diff
@@ -0,0 +1,63 @@
+# General information on Privacy
+
+## About transaction graphs
+
+It’s important to understand, that transactions are pseudonymous, but not anonymous. That means, a transaction graph can be built up, and there exists companies that are doing that work. If at some point one of our transactions is connected to our real world identity, it becomes easy to make assumptions based on that, for example that other transactions also belong to us, or that a specific unspent transaction belongs to us (that could give information about our balance). 
+By  checking the status of our transaction on a blockchain explorer, our IP address can be connected to one of our wallet addresses, this could create a starting point for a transaction graph. A possible solution for this is to use blockchain explorer only over Tor.
+
+## Transaction broadcasting
+
+Broadcasting the transaction can reveal that the transaction you are broadcasting belongs to you. A solution for this is to broadcast your transaction through a website, over Tor. For example, [https://explorer.dcrdata.org/decodetx](https://explorer.dcrdata.org/decodetx)
+It's also important to note, that different wallets might use different [address types](address-details). This can also give a hint about your identity, which combined with other signals can ultimately reveal your identity.
+
+## Trade-offs of reusing vs not reusing addresses
+
+For privacy, the best practice is to not reuse an address ever. A cryptocurrency address can be thought of as a cryptocurrency invoice instead. The term "address" does not mean, that it was designed to reuse it, like one would do with a bank account number for example. If someone would really care about privacy he or she would use a new receiving address even on a website, generated by a script. 
+On the other hand, address reuse is preferred in situations when privacy is not important and we don't want to make the mistake of sending over large amount of funds to a wrong address. Reusing addresses is also preferred when transparency is important.
+For POS voting, not using the same address is not possible yet. 
```

Comment:
I would remove this line about PoS voting because its not strictly true. Also it will become redundant when privacy is implemented into VSPs